packages/clipper/src/domtoimage.ts

```diff
@@ -202,7 +186,13 @@ const VALID_ATTRIBUTES = [
   "width",
   "height",
   "target",
-  "rel"
+  "rel",
+  "class",
+  "id",
+  "disabled",
+  "type",
+  "data-",
+  "aria-"
```

Comment:
This might not be an exhaustive list of attributes that could be used in CSS. What is the significance of the valid attribute check?

Reply:
I think this was added to trim down on useless attributes since we were already copying the styles to each element. Since that is no longer the case, we can remove this.